_posts/en/newsletters/2020-09-30-newsletter.md

```diff
@@ -54,6 +56,35 @@ popular Bitcoin infrastructure software.
     problems that are the result of this bug---but developers are still
     investigating.
 
+- **US Patent 7,110,538 has expired:** Bitcoin transactions are secured using
+  [ECDSA][] (the _Elliptic Curve Digital Signature Algorithm_). Verifying
+  signatures involves multiplying points on the elliptic curve by scalars.
+  Typically, each transaction input requires one or more signature verifications,
+  meaning that syncing the Bitcoin block chain requires many millions of these
```

Comment:
Maybe s/requires/can require/ since signature verification is disabled in assumedvalid blocks by default.

Reply:
done